src/libs/httpserver/indexDataFetcher.ts

```diff
@@ -15,38 +30,29 @@ export const fetchAppDataForSlug = async (slug, client) => {
     const { Cookie: cookie, ...cozyDataAttributes } =
       cozyDataResult.data.attributes
 
-    const cozyData = {
+    const cozyData: CozyData = {
       app: {
         editor: cozyDataAttributes.AppEditor,
         icon: cozyDataAttributes.IconPath,
         name: cozyDataAttributes.AppName,
         prefix: cozyDataAttributes.AppNamePrefix,
         slug: cozyDataAttributes.AppSlug
       },
-      capabilities: JSON.parse(cozyDataAttributes.Capabilities),
+      capabilities: JSON.parse(cozyDataAttributes.Capabilities) as Record<
+        string,
+        boolean
+      >,
       domain: cozyDataAttributes.Domain,
-      flags: JSON.parse(cozyDataAttributes.Flags),
+      flags: JSON.parse(cozyDataAttributes.Flags) as Record<string, boolean>,
       locale: cozyDataAttributes.Locale,
-      subdomain: cozyDataAttributes.SubDomain,
+      subdomain: cozyDataAttributes.SubDomain as 'nested' | 'flat',
       token: cozyDataAttributes.Token,
       tracking: cozyDataAttributes.Tracking
     }
 
     let cozyDataString = JSON.stringify(cozyData)
     cozyDataString = replaceAll(cozyDataString, '"', '&#34;')
 
-    cozyDataAttributes.Flags = replaceAll(
```

Comment:
I remember this hack and removed it, somehow I didn't see any issue in the produced HTML

Reply:
Are you sure this isn't needed anymore? 🤔

Reply:
Yeah it seems so, because the stack changed the way strings are formatted. I couldn't reproduce any issue with removing all the formatting we did before

Reply:
> Yeah it seems so, because the stack changed the way strings are formatted. 

This is not something I'm aware of. @nono, does the stack changed something about cozy-data.flags encoding?

Reply:
> > Yeah it seems so, because the stack changed the way strings are formatted.
> 
> This is not something I'm aware of. @nono, does the stack changed something about cozy-data.flags encoding?

I seemed certain about that but it's just an observation. Seems like what we get from the stack is escaped correctly and we do not need manipulation like before. Or maybe it was like that from the start? Or maybe I'm wrong, but from my end, the cozy-data and the other attributes are all correctly injected into this PR

Reply:
I don't think the stack has changed something about that. Maybe, when upgrading the go version used to compile the code, a change in the stdlib has an effect on that.